### PR TITLE
Added completions endpoint/6.5.0

### PR DESCRIPTION
**Pull request by tripathiarpan20**

### CODE REVIEW

File: core/create_config.py [hunk 1]
```diff
@@ -81,8 +81,11 @@ def generate_miner_config(dev: bool = False) -> dict[str, Any]:
     config["NODE_PORT"] = input("Enter the port to run the miner server on (default: 4001): ") or "4001"
     config["IMAGE_WORKER_URL"] = input("Enter IMAGE_WORKER_URL: ")
     config["LLAMA_3_1_8B_TEXT_WORKER_URL"] = input("Enter LLAMA_3_1_8B_TEXT_WORKER_URL: ")
+    config["LLAMA_3_1_8B_TEXT_COMP_WORKER_URL"] = input("Enter LLAMA_3_1_8B_TEXT_COMP_WORKER_URL: ")
```
Comment (namoray) [inline core/create_config.py:84]: They don't need different urls, same gpu

File: core/task_config.py [hunk 2]
```diff
@@ -44,14 +49,41 @@ def task_configs_factory() -> dict[str, cmodels.FullTaskConfig]:
             endpoint=cmodels.Endpoints.chat_completions.value,
             volume_to_requests_conversion=300,
             is_stream=True,
-            weight=0.05,
+            weight=0.025,
+            timeout=2,
+            enabled=True,
+        ),
+        CHAT_LLAMA_3_2_3B_COMP: cmodels.FullTaskConfig(
```
Comment (namoray) [inline core/task_config.py:56]: Should these be different tasks? We don't want people to support one endpoint and not the other right? Plus nearly all the config is duplicated here. They should have the same quality score for both imo, not sure it makes sense for them to be separate

File: validator/entry_node/src/endpoints/text.py [hunk 3]
```diff
@@ -182,3 +229,11 @@ async def chat(
     response_model=None,
     dependencies=[Depends(verify_api_key_rate_limit)],
 )
+router.add_api_route(
+    "/v1/completions",
+    chat,
```
Comment (namoray) [inline validator/entry_node/src/endpoints/text.py:234]: shouldn't this be chat_comp?

File: core/task_config.py [hunk 4]
```diff
@@ -44,14 +49,41 @@ def task_configs_factory() -> dict[str, cmodels.FullTaskConfig]:
...
+            orchestrator_server_config=cmodels.OrchestratorServerConfig(
+                server_needed=cmodels.ServerType.LLM,
+                load_model_config={
+                    "model": "unsloth/Llama-3.2-3B-Instruct",
+                    "half_precision": True,
+                    "tokenizer": "tau-vision/llama-tokenizer-fix",
+                    "max_model_len": 20_000,
+                    "gpu_utilization": 0.5,
+                },
+                endpoint=cmodels.Endpoints.completions.value,
```
Comment (sirouk) [inline core/task_config.py:69]: This is what differs right? @namoray if folks are to serve both endpoints, what do we do with the endpoint? Should it become a list, or tuple?

File: core/task_config.py [hunk 5]
```diff
@@ -44,14 +49,41 @@ def task_configs_factory() -> dict[str, cmodels.FullTaskConfig]:
...
+                checking_function="check_text_result",
+                task=CHAT_LLAMA_3_2_3B_COMP,
+            ),
+            synthetic_generation_config=cmodels.SyntheticGenerationConfig(
+                func="generate_chat_comp_synthetic", kwargs={"model": CHAT_LLAMA_3_2_3B_COMP}
+            ),
+            endpoint=cmodels.Endpoints.completions.value,
```
Comment (sirouk) [inline core/task_config.py:76]: @namoray the same question (directly above) goes for this

**Pull request merged**